frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card-body.tsx

```diff
@@ -76,31 +56,30 @@ export const CapacityCardBody: React.FC<CapacityCardBodyProps> = ({ isLoading, m
     );
   }
   return (
-    <div>
-      <span className="text-secondary">
-        <h4>Usage Breakdown</h4>
-      </span>
-      <ChartDonut
-        donutHeight={250}
-        donutWidth={110}
-        labelPosition="startAngle"
-        legendPosition="right"
-        legendData={metricLegend}
-        legendOrientation="vertical"
-        data={metricData}
-        labels={(datum) => `${datum.x}: ${datum.y.toFixed(2)}%`}
-        subTitlePosition="bottom"
-        themeColor={ChartThemeColor.multi}
-        themeVariant={ChartThemeVariant.light}
-        title={humanizeBinaryBytesWithoutB(total).string}
-        radius={53}
-        innerRadius={46}
-      />
+    <div className="nb-capacity-card__body">
```

Comment:
can we rename it to `nb-capacity-card__chart-body` to not confuse with the css for`<CapacityCardBody>`

Reply:
Done...